starter-code/ShoppingListWithDetailView/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistdetailview/FirstFragment.java

```diff
@@ -0,0 +1,57 @@
+package ly.generalassemb.drewmahrt.shoppinglistdetailview;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.List;
+
+/**
+ * Created by Serkan on 01/11/16.
+ */
+
+public class FirstFragment extends Fragment {
```

Comment:
I suggest calling this ListFragment rather than FirstFragment, so it's more clear which feature this is part of